main.tf

```diff
@@ -141,13 +130,14 @@ locals {
   })
 }
 
-resource "aws_autoscaling_group" "default" {
+resource "aws_autoscaling_group" "this" {
   count = var.enabled ? 1 : 0
 
-  name_prefix               = format("%s%s", module.label.id, var.delimiter)
+  name                      = var.name
   vpc_zone_identifier       = var.subnet_ids
   max_size                  = var.max_size
   min_size                  = var.min_size
+  desired_capacity          = var.desired_capacity
```

Comment:
If this is the main change, can we:

1. Boil it down to a minimal patch
2. File a public PR
3. Use our fork until step 2 is either accepted or provides a work-around

?